pallets/capacity/src/lib.rs

```diff
@@ -528,7 +528,11 @@ impl<T: Config> Pallet<T> {
 		target: MessageSourceId,
 		target_details: StakingTargetDetails<BalanceOf<T>>,
 	) {
-		StakingTargetLedger::<T>::insert(staker, target, target_details);
+		if target_details.amount.is_zero() {
```

Comment:
this was an oversight - if they've unstaked everything this should be removed.

Reply:
Vaguely remember Wil or someone issued a similar fix in frequency, maybe they are unrelated

Reply:
Yes that was unrelated 
https://github.com/LibertyDSNP/frequency/pull/1769/files